draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1969,13 +2159,13 @@ The verifier would decode the HS-Disclosures JWT and SD-JWT as follows:
    *  Editorial fixes
    *  Added `hash_alg` claim
    *  Renamed `_sd` to `sd_digests` and `sd_release`
-   *  Added descriptions on holder binding - more work to do
+   *  Added descriptions on Holder Binding - more work to do
    *  Clarify that signing the SD-JWT is mandatory
 
    -00
 
    *  Renamed to SD-JWT (focus on JWT instead of JWS since signature is optional)
-   *  Make holder binding optional
+   *  Make Holder Binding optional
    *  Rename proof to release, since when there is no signature, the term "proof" can be misleading
```

Comment:
```suggestion
   *  Made Holder Binding optional
   *  Renamed proof to release, since when there is no signature, the term "proof" can be misleading
```